testing/packer/APKBUILD

```diff
@@ -0,0 +1,39 @@
+# Contributor: 7heo <7heo@mail.com>
+# Maintainer: 7heo <7heo@mail.com>
+pkgname=packer
+pkgver=0.10.2
+pkgrel=0
+pkgdesc="Packer is a tool for creating identical machine images for multiple platforms from a single source configuration."
+url="https://www.packer.io/"
+arch="x86_64"
+license="MPL 2.0"
+depends=""
+depends_dev=""
+makedepends="$depends_dev go go-tools"
```

Comment:
This package doesn’t provide -dev subpackage, so `depends_dev` should not be declared here, nor used in `makedepends`. aport tool even warns about this.